tests/test_crypto.py

```diff
@@ -69,3 +81,98 @@ def test_subprocess_raises_exception(safe_tmpdir, mocker):
         gpg.decrypt_submission_or_reply(test_gzip, output_filename, is_doc=True)
 
     assert mock_gpg.call_count == 1
+
+
+def test_import_key(safe_tmpdir, config, source):
+    '''
+    Using the `config` fixture to ensture the config is written to disk.
```

Comment:
Nit: typo on ensure (same for the rest of the docstrings in this file)

Reply:
ah crap :neutral_face: :neutral_face: